### DISCUSSION TITLE
Unable to install on Ubuntu 22.04.2 LTS

### DISCUSSION THREAD

**Discussion by Craigpgardner** [amidaware/tacticalrmm#1495] [Q&A]
I just tried to install on Ubuntu 22.04.2 LTS and recieved an error stating "Supported versions: ubuntu 20.04, Debian 10 and 11 Your system does nto appear to be supported. 

I thought we could install on the latest ubuntu release? Any advice/assistance would be greatly appreciated. 

Thank you,

**Comment by dinger1986** [comment]
Ubuntu 20.04 and Debian 10 and 11 for the server.

The agent can be installed on any modern linux distro

**Comment by Craigpgardner** [comment]
Thank you!
